### DISCUSSION TITLE
Account Flagged Please Help

### DISCUSSION THREAD

**Discussion by Marioacm1208** [community/community#66962] [Other Feature Feedback, Questions, & Ideas]
Account Flagged

Question

### Body

I think GitHub is flagging my account for no reason, I've opened a ticket more than a week ago and nobody in support seems to care. I need to get back to work, I'm paying for my organization's GitHub Copilot and I'm losing my benefits since now I cannot even accept invitations to organizations. (Yes dumb me exited from organization as part of troubleshooting steps). I'm very desperate and frustrated at this point. What I need to do to get my account back to normal?

Ticket #2319458

**Comment by Marioacm1208** [comment]
![image](https://github.com/community/community/assets/17240923/1b9ff92e-2ed8-4f32-8086-088b4fc2694e)

**Comment by Marioacm1208** [comment]
![image](https://github.com/community/community/assets/17240923/e6692a01-39d6-47a4-ae3a-5b3f5a99bb06)

**Reply by Marioacm1208** [reply]
Already did... No resolution, 8 days later

**Comment by pipinthedev** [comment]
still have issue ?
